### ISSUE TITLE
Jekyll Build fail on travis-ci

### ISSUE THREAD

**Issue by krichprollsch** [clermontech/planet#149]
Commit e1e3cd1d444bee2b6eadd8d88f205abde25c0406 introduced a build error with Travis-ci

```bash
ERROR: YOUR SITE COULD NOT BE BUILT:
 ------------------------------------
Invalid date '0000-00-00': Post '/vendor/bundle/ruby/1.9.1/gems/jekyll-2.4.0/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the filename.
```

https://travis-ci.org/clermontech/planet/builds/50185506

**Comment by willdurand** [comment]
Fixed by 353e0b7cb1e4809f42c01e37c2a965890fc11ba7

**Comment by krichprollsch** [comment]
thanks !
